_includes/payment-order-purchase.md

```diff
@@ -268,3 +268,7 @@ Content-Type: application/json
 
 [one-click-payments]: /payment-menu/other-features#one-click-payments
 [settlement-and-reconciliation]: /payment-menu/other-features#settlement-and-reconciliation
+[initiate-consumer-session]: /checkout/checkin#step-1-initiate-session-for-consumer-identification
+[user-agent]: https://en.wikipedia.org/wiki/User_agent
+[operations]: /checkout/other-features#operations
+[split-settlement]: /payment-menu/other-features#split-settlement
```

Comment:
We order our links alphabetically, you can easily fix this by using the function `Sort Lines Ascending` in Visual Studio Code. 😃